reference/services/mirror-prober.rst

```diff
@@ -0,0 +1,32 @@
+Mirror prober
+=============
+
+Description
+-----------
+
+The mirror prober script checks the Ubuntu `CD mirrors`_ and the
+`archive mirrors`_ configured in Launchpad and updates their status based on
+the results of the health checks. It is run via a cron job on the
+``launchpad-scripts`` unit.
+
+.. _CD mirrors: https://launchpad.net/ubuntu/+cdmirrors
+.. _archive mirrors: https://launchpad.net/ubuntu/+archivemirrors
+
+Log files
+---------
+
+The ``{{ logs_dir }}/mirror-prober.log`` file (where ``logs_dir`` is usually
+``/srv/launchpad/logs``) contains the generic logs for each run of the mirror
+prober.
```

Comment:
What it be possible to have a similar formatted section like in https://documentation.ubuntu.com/launchpad/en/latest/reference/services/signing/#log-files ?

Reply:
In the referenced section log files are presented in a way which can be used very easily, e.g. you can read:

```
rless il3-signing1.lp.internal::lp-signing-gunicorn-logs/gunicorn.log
```

I can log into the relevant box, and copy/paste that command.

Reading only the above section "Log files", you need a lot of background knowledge, e.g.
- I do not see which machine is relevant, except I scroll up and read previous sections
- When I scroll up, I see `launchpad-scripts` unit is mentioned - without context, I would not know how I can log into it.
- Then I still do not know the path to the log. Saying "where ``logs_dir`` is usually
``/srv/launchpad/logs``" is a bit confusing... as there are no examples, I do not know how many different logs I can expect. Is it one log per mirror, or one log for all mirrors? If the former, why are they not all in the same directory, and if they are not in the same directory, where are they?

This can be all clarified by actually logging into the host and investigating, but I would hope we can make those kind of "first aid" documentation pages more succinct.

So all in all, I would love to have
- a step by step instruction on how to log into the launchpad-scripts unit
- depending on the log format (one for all or one for each of the mirrors) a clear path to the log(s)
- no vagueness about where the logs are (if that is possible - if that is not possible, there should be some more context)